packages/x-components/src/x-modules/search-box/wiring.ts

```diff
@@ -46,20 +50,36 @@ const clearSearchBoxQuery = wireCommit('setQuery', '');
  */
 const setUrlParams = wireDispatch('setUrlParams');
 
+/**
+ * Sets the search state `status`.
+ *
+ * @param event - The {@link XEvent} used to transition the status.
+ *
+ * @returns A wire.
+ *
+ * @public
+ */
+const setInputStatus = (event: XEvent): AnyWire => wireDispatch('setInputStatus', event);
+
 /**
  * SearchBox wiring.
  *
  * @internal
  */
 export const searchBoxWiring = createWiring({
   ParamsLoadedFromUrl: {
-    setUrlParams
+    setUrlParams,
+    transitionState: filter(setInputStatus('UserAcceptedAQuery'), ({ eventPayload: urlParams }) => {
```

Comment:
Hmm smells like ñapa... I don't think this event re-mapping is a good idea... Probably the url module should emit the `UserAcceptedAQuery` event itself somehow... But not for this PR